packages/rum/src/domain/record/observers.spec.ts

```diff
@@ -50,6 +55,16 @@ describe('initInputObserver', () => {
     })
   })
 
+  it('collects input values when an "input" event is composed', () => {
+    stopInputObserver = initInputObserver(inputCallbackSpy, DefaultPrivacyLevel.ALLOW)
+    dispatchComposedInputEvent('foo')
+
+    expect(inputCallbackSpy).toHaveBeenCalledOnceWith({
+      text: 'foo',
+      id: jasmine.any(Number) as unknown as number,
+    })
+  })
```

Comment:
I cannot test with a shadow DOM because it seems that untrusted event do not go through. I create fake composed event and it _works_

Reply:
Could you add a comment about that in the file, so we don't struggle again when we try to dispatch events in shadow DOM in the future?